### PR TITLE
config: add signet network option

### PR DESCRIPTION
**Pull request by ellemouton**
Allow `--network=signet`

### CODE REVIEW

**Review by guggero — APPROVED** [review]
utACK, LGTM :tada:

**Review by dstadulis — APPROVED** [review]
ACK on 850e6fa077e1c1ed692222b440609ea3a629b938

Are any release notes helpful to caveat signet performance with the liquidity-service for Loop/Pool etc?

**Review by ViktorTigerstrom — APPROVED** [review]
utACK LGTM :rocket:!

We have this issue for `regtest` as well:
But should we mention somewhere that `autopilot` should be disabled if running on `signet`, as we don't support it on the autopilot server yet? 
Maybe that's better suited for `docs.lightning.engineering` though.

**Comment by ellemouton** [comment]
> Maybe that's better suited for docs.lightning.engineering though.

Yeah i think that is ok. Cause LiT will already throw an error if a user tries to spin up LiT on regtest/signet with autopilot enabled

File: config.go [hunk 1]
```diff
@@ -179,7 +179,7 @@ type Config struct {
 	// before the configuration is loaded and will set the correct flag on
 	// `lnd.bitcoin.mainnet|testnet|regtest` and also for the other daemons.
```
[the remark this answers is not in the document]
Comment (ellemouton) [reply config.go]: good catch, thanks! fixed

**Pull request merged**